specification/network/resource-manager/Microsoft.Network/stable/2024-01-01/bastionHost.json

```diff
@@ -1103,6 +1233,16 @@
       },
       "description": "The list of sessionids to disconnect.",
       "x-ms-parameter-location": "method"
+    },
+    "SasUrl": {
+      "name": "sasurl",
+      "in": "body",
+      "required": true,
+      "schema": {
+        "type": "string"
```

Comment:
I strongly recommend using an object type schema instead of a RAW string, otherwise you'll find this API very difficult to add a new improved version!

Reply:
Per other comment, the use of this API is quite limited so planning on keeping it like this for now

Reply:
Please change this to a JSON payload per other comments and the RPC.